pkg/machine/wsl/machine.go

```diff
@@ -660,9 +660,13 @@ func getAllWSLDistros(running bool) (map[string]struct{}, error) {
 	return all, nil
 }
 
-func isSystemdRunning(dist string) (bool, error) {
+func isSystemdRunning(dist string, podmanSetup bool) (bool, error) {
 	cmd := exec.Command(wutil.FindWSL(), "-u", "root", "-d", dist, "sh")
-	cmd.Stdin = strings.NewReader(sysdpid + "\necho $SYSDPID\n")
+	sysdpidScript := sysdpid
+	if !podmanSetup {
+		sysdpidScript = sysdpidSystemdConfig
```

Comment:
Not sure how important it is, we configured the VM to start with systemd, so if it’s not there we have a big problem. But we also check that wsl  is running in `isRunning`, so I guess it’s more a sanity check than something which will actually happen?

Reply:
yes if you prefer we could also remove it i guess. I wanted to keep the code logic as it is now but maybe it is superfluous if we enable systemd through the wsl.conf. If it does not run for some reason i expect the distro to not be running, so checking its state could be enough. Let me know what you prefer